### DISCUSSION TITLE
Redesign of the Elexis splash image

### DISCUSSION THREAD

**Discussion by debenol1** [elexis/elexis.github.io#6] [Announcements]
The current Elexis splash image is to be redesigned. The following points have been improved:
- Reduction in the pixelation of the lettering
- Improvement in readability by increasing the contrast between the lettering and the background motif
- Standardization of the color space

The new splash image was designed as follows:
- Type area with the following ratio: a/a+b = b/a (golden ratio)
- Vectorization of the lettering
- Increased transparency of the mountain motif
- Antagonists of the original violet derived using the color wheel (complementary/split)

This results in 14 variants, which are presented below. To ensure that the decision for the successor splash image is transparent, a [poll](https://github.com/orgs/elexis/discussions/8) is being launched

**Comment by debenol1** [comment]
**#1**

![splash-image-1](https://github.com/user-attachments/assets/e22f69a8-078a-47ff-a0fd-092d1986435d)

**Comment by debenol1** [comment]
**# 2**

![splash-image-2](https://github.com/user-attachments/assets/af88c906-7f90-4bcd-a135-18b270d3f39b)

**Comment by debenol1** [comment]
**# 3**

![splash-image-3](https://github.com/user-attachments/assets/9e0326d0-82ed-493a-87ef-2c8b3ce2edef)

**Comment by debenol1** [comment]
**# 4**

![splash-image-4](https://github.com/user-attachments/assets/ebb760c7-bd94-4215-bf93-a1cd4c1159e0)

**Comment by debenol1** [comment]
**# 5**

![splash-image-5](https://github.com/user-attachments/assets/2c1e5b5b-05c8-4033-9cb9-708e88baa402)

**Comment by debenol1** [comment]
**# 6**

![splash-image-6](https://github.com/user-attachments/assets/0e3f89f2-d077-4ca2-a8fc-af7a4e364e02)

**Reply by col-panic** [reply]
Heißt das 8l8xis? :)

**Comment by debenol1** [comment]
**# 7**

![splash-image-7](https://github.com/user-attachments/assets/360b42fb-5913-40f0-ab8a-70501bec8a96)

**Comment by debenol1** [comment]
**# 8**

![splash-image-8](https://github.com/user-attachments/assets/dc49ae3b-f746-4aa3-86e6-c1de81591253)

**Comment by debenol1** [comment]
**# 9**

![splash-image-9](https://github.com/user-attachments/assets/5708a1b5-c297-41af-80fa-b7a831e6667a)

**Comment by debenol1** [comment]
**# 10**

![splash-image-10](https://github.com/user-attachments/assets/ac974755-147f-4016-8a02-f5d30d47e394)

**Comment by debenol1** [comment]
**# 11**

![splash-image-11](https://github.com/user-attachments/assets/76b6c8d1-cea4-4032-8095-6d2b410621e4)

**Comment by debenol1** [comment]
**# 12**

![splash-image-12](https://github.com/user-attachments/assets/96b96e84-4162-4d04-a8ce-87ee197e4741)

**Comment by debenol1** [comment]
**# 13**

![splash-image-13](https://github.com/user-attachments/assets/97029246-6722-48f8-8b65-27b42f7746ee)

**Comment by debenol1** [comment]
**# 14**

![splash-image-14](https://github.com/user-attachments/assets/b7496b6a-e260-4cf0-99a2-7b6333bbb856)

**Comment by debenol1** [comment]
According the feedbacks #6 will be the new splash image

**Comment by col-panic** [comment]
@debenol1 is it already integrated?
